README.md

```diff
@@ -633,6 +614,26 @@
     }
     ```
 
+    Use the same pattern to create larger strings that won't fit in one line, since string concatenation for larger strings can impact performance. [jsPerf](http://jsperf.com/ya-string-concat) & [Discussion](https://github.com/airbnb/javascript/issues/40)
```

Comment:
Condensed the suggestion of the use of Array#join here instead of handling long strings on a different way when doing concatenation.